contribute.json

```diff
@@ -0,0 +1,40 @@
+{
+  "name": "Phabricator",
+  "description": "Mozilla's Phabricator instance for code review.",
+  "repository": {
+    "url": "https://github.com/mozilla-services/mozphab",
+    "license": "MPL2",
+    "tests": "https://circleci.com/gh/mozilla-services/mozphab/tree/master"
+  },
+  "participate": {
+    "home": "https://moz-conduit.readthedocs.io/en/latest/",
+    "docs": "https://moz-conduit.readthedocs.io/en/latest/",
+    "mailing-list": "https://www.mozilla.org/en-US/about/forums/#tools",
+    "irc": "irc://irc.mozilla.org/#phabricator",
+    "irc-contacts": [
+      "mcote",
+      "dkl",
+      "ckolos"
+    ]
+  },
+  "bugs": {
+    "list": "https://bugzilla.mozilla.org/describecomponents.cgi?product=Conduit&component=Phabricator%20Extensions#Phabricator%20Extensions",
+    "report": "https://bugzilla.mozilla.org/enter_bug.cgi?product=Conduit&component=Phabricator%20(Upstream)",
+    "mentored": "https://bugzilla.mozilla.org/buglist.cgi?list_id=13755353&resolution=---&query_format=advanced&component=Phabricator%20%28Upstream%29&component=Phabricator%20Extensions&product=Conduit"
+  },
+  "urls": {
+    "prod": "https://phabricator.services.mozilla.com/",
+    "stage": "https://phabricator.allizom.org",
+    "dev": "https://phabricator-dev.allizom.org"
+  },
+  "keywords": [
+    "phabricator",
+    "php",
+    "code review",
```

Comment:
I have a feeling these should be single words, maybe separated by a hyphen if necessary.